Models/models.js

```diff
@@ -9,25 +9,45 @@ exports.selectCategories = () => {
     })
 }
 
-exports.selectReviews = () => {
-    return db.query(
+exports.selectReviews = (order = 'DESC', sort_by = 'created_at', category) => {
+    if (!['ASC', 'DESC'].includes(order.toUpperCase())){
+        order = 'DESC'
```

Comment:
The default should only apply if the query itself isn't applied to the request - otherwise it would be best to respond with a 400 - otherwise the client could be under the impression their data is sorted by something the provided that is actually invalid